features/auth/signup/BasicForm.tsx

```diff
@@ -116,6 +98,26 @@ export default function BasicForm({
           helperText={errors?.email?.message ?? " "}
           error={!!errors?.email?.message}
         />
+        <InputLabel className={authClasses.formLabel} htmlFor="password">
+          Password
+        </InputLabel>
+        <TextField
+          id="password"
+          fullWidth
+          className={authClasses.formField}
+          name="password"
+          type="password"
+          variant="standard"
+          inputRef={register({
+            pattern: {
+              message: "Please enter a password",
+              value: passwordValidationPatern,
+            },
+            required: "Password can't be empty.",
+          })}
+          helperText={errors?.password?.message ?? " "}
+          error={!!errors?.password?.message}
```

Comment:
We shouldn't be hardcoding messages directly in the component anymore as migrating hardcoded messages/constants to translation resources to allow the project to be localised has been painful. And if we leave it now, I'm almost certain we'll forget this and give ourselves with more work to do 😅  so I'd recommend putting that into the resource file and use the `t` function like the surrounding code does - you only need to populate the English resource!